CHANGELOG.md

```diff
@@ -8,6 +8,50 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.0] - 2002-12-12
```

Comment:
Please add the link to the footer list below and update the Unreleased link in the footer as well.